lib/inc/hocon/config.hpp

```diff
@@ -570,6 +570,9 @@ namespace hocon {
         virtual unwrapped_value get_any_ref(std::string const& path) const;
         virtual std::shared_ptr<const config_value> get_value(std::string const& path) const;
 
+        template<typename T>
+        std::vector<T> get_homogeneous_unwrapped_list(std::string const& path) const;
```

Comment:
Why make this public? I don't see any public use of it, and to create new templated forms the full function would need to be in the header file.